es-ds-components/components/es-cta-card.vue

```diff
@@ -0,0 +1,60 @@
+<script setup>
+const props = defineProps({
+    imageFirst: {
+        type: Boolean,
+        default: false,
+    },
+    horizontalPadding: {
+        type: String,
+        default: 'px-100',
+    },
+    verticalSpacing: {
+        type: String,
+        default: 'mb-150',
+    },
+});
+</script>
+
+<template>
+    <es-card
+        class="align-items-center d-flex flex-column py-200 text-center"
+        :class="props.horizontalPadding"
+        v-bind="$attrs">
+        <!-- avoiding use of an <h2> tag here for long-form content SEO reasons,
+            but preserving heading semantics for screen readers -->
+        <div
+            role="heading"
+            aria-level="2"
+            class="font-weight-semibold font-size-300"
+            :class="[{ 'order-1': props.imageFirst }, props.verticalSpacing]">
+            <slot name="heading"> Easily find what solar costs in your area </slot>
+        </div>
+        <div
+            v-if="$slots.image"
+            class="mx-auto w-100"
+            :class="props.verticalSpacing">
+            <slot name="image" />
+        </div>
+        <div
+            class="w-100"
+            :class="{ 'order-2': props.imageFirst }">
+            <div
+                v-if="$slots.subtitle"
+                class="es-cta-card-subtitle"
+                :class="props.verticalSpacing">
+                <slot name="subtitle" />
+            </div>
+            <slot name="cta" />
+        </div>
+    </es-card>
+</template>
+
+<style lang="scss" scoped>
+.es-cta-card {
+    &-subtitle {
+        ::v-deep p:last-child {
```

Comment:
I got this warning in the console - `::v-deep usage as a combinator has been deprecated. Use :deep(<inner-selector>) instead of ::v-deep <inner-selector>.`

Reply:
Oops yep - fixed - thanks!